docs/technical-documentation.md

```diff
@@ -93,27 +93,28 @@ import "./lib/WithdrawLockLib.sol";
+// example: the app's standard `lockDuration` multiplied by the user's active locks multiplied by the `spamPenaltyFactor` on this lock, all divided by PCT_BASE to either create a multiplier or fractional percentage of the standard `lockDuration`
 // - note: since this is a constant it can be set here rather than in the `initialize()` function
-uint256 private constant WHOLE_GRIEFING = 100;
+// The spam penalty value is expressed between zero and a maximum of 10^18 (that represents 100%). As a consequence, it's important to consider that 1% is actually represented by 10^16.
+uint256 public constant PCT_BASE = 10 ** 18;
 ```
 
-### Griefing Variables Explained
+### Spam penalty Variables Explained
 
-The griefing calculation does not reflect how many tokens should be locked and for how long, but rather the amount and duration to add to the base lockAmount and lockDuration. The `griefingFactor` is a % of the base lock amount and duration values set on the app. This value increases the more locks an account has, making it more and more expensive to create many locks.
+The spam penalty calculation does not reflect how many tokens should be locked and for how long, but rather the amount and duration to add to the base lockAmount and lockDuration. The `spamPenaltyFactor` is a % of the base lock amount and duration values set on the app. This value increases the more locks an account has, making it more and more expensive to create many locks.
 
-When an account wants to submit a proposal they will have to lock `lockAmount` + (`lockAmount` _ `totalActiveLocks` _ `griefingFactor`/ `WHOLE_GRIEFING`) for a duration of `lockDuration` + (`lockDuration` _ `totalActiveLocks` _ `griefingFactor` / `WHOLE_GRIEFING`)
+When an account wants to submit a proposal they will have to lock `lockAmount` + (`lockAmount` _ `totalActiveLocks` _ `spamPenaltyFactor`/ `PCT_BASE`) for a duration of `lockDuration` + (`lockDuration` _ `totalActiveLocks` _ `spamPenaltyFactor` / `PCT_BASE`)
 
-e.g. if the `lockAmount` = 20 tokens, `lockDuration` = 6 days and `griefingFactor` is 50%, and the account submitting a proposal has 2 active locks, they will have to lock 40 tokens for 12 days.
+e.g. if the `lockAmount` = 20 tokens, `lockDuration` = 6 days and `spamPenaltyFactor` is 50%, and the account submitting a proposal has 2 active locks, they will have to lock 40 tokens for 12 days.
 
 The idea behind this is to prevent spamming of proposals.
 
-> Note: this only works if permissions on the lock-app are set so that only members of the DAO `canForward()`. If _anyone_ can submit proposals or DAO members can easily transfer their membership tokens between accounts the griefing mechanism is much less effective.
+> Note: this only works if permissions on the lock-app are set so that only members of the DAO `canForward()`. If _anyone_ can submit proposals or DAO members can easily transfer their membership tokens between accounts the spam penalty mechanism is much less effective.
```

Comment:
```suggestion
> Note: this only works if permissions on the time lock app are set so that only members of the DAO `canForward()`. If _anyone_ can submit proposals or DAO members can easily transfer their membership tokens between accounts the spam penalty mechanism is much less effective.